FogCreek/shared-components
# Sour environment: remote component

Demo: http://sour-environment.glitch.me/test/remote-component

How to test in your app:
- install `"@fogcreek/shared-components": "0.1.1-4382413.0"`
- remix `sour-environment.glitch.me`
- add an icon to your remix's version of `lib/icon.js`
- import this updated icon component with 
```js
import { createRemoteComponent } from '@fogcreek/shared-components'
const Icon = createRemoteComponent('https://<your remix>.glitch.me/module.js', 'Icon');`
```
- render `<Icon icon="<your new icon>" />`
- it should display the icon you just added
